src/homepageExperience/components/steps/cli/InitializeClient.tsx

```diff
@@ -104,14 +104,14 @@ export const InitializeClient: FC<OwnProps> = ({
       dispatch(createAuthorization(authorization))
       event(`firstMile.${wizardEventName}.tokens.tokenCreated`)
     }
-  }, [sortedPermissionTypes.length])
+  }, [sortedPermissionTypes.length, tokenValue]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Same comment here. Unless there's a reason I would add all missing dependencies to the array, and not do `// eslint-disable-line`